readme.md

```diff
@@ -31,33 +31,43 @@ TODO:
 
 ## Installation
 
-`python3.8` required.
-
-
-From pip:
+### From PyPI
 
 ```sh
 pip3 install tg
+tg
 ```
 
-From sources:
+### Using the GitHub sources 
+
+> Requires [flit](https://github.com/takluyver/flit) to be installed.
+>
+> Install it with:
+> ```sh 
+> pip3 install flit
+> ```
 
 ```sh
-pip3 install python-telegram
 git clone git@github.com:paul-nameless/tg.git
 cd tg
-PYTHONPATH=. python3 tg/main.py
```

Comment:
i think it is still worth to mention about ` pip3 install python-telegram & python3 -m tg` method. because you don't required to install `flit` to use tg client

Reply:
I'm not sure that mixing package managers would be a great idea, but I can add that anyways.

I was thinking of recommending the creation of a virtualenv, is that ok for you?